index.js

```diff
@@ -208,7 +217,7 @@ const generateRollupConfig = function(options) {
       // but only if the plugin is a string and not a
       // primed plugin already.
       plugins: buildSettings.plugins[buildType].map((p) => typeof p !== 'string' ? p : buildSettings.primedPlugins[p]),
-      external: buildSettings.externals[buildType],
+      external: (id) => buildSettings.externals[buildType].some((ext) => id.startsWith(ext)),
```

Comment:
allows external to specify just the main module as external, rather than having to do it for each require. IE `mux.js` vs `mux.js/lib/mp4/transmuxer` and `mux.js/lib/mp4/caption-parser`